lib/MetagenomeAPI/MetagenomeSearchUtils.py

```diff
@@ -0,0 +1,151 @@
+# -*- coding: utf-8 -*-
+import json
+import os
+import requests
+import time
+
+
+class MetagenomeSearchUtils:
+
+    def __init__(self, config):
+        if config.get('elastic-url'):
```

Comment:
I think you also have `os.environ['KBASE_ENDPOINT']` which would be something like `https://ci.kbase.us/services`, maybe that's easier to use here